author_api/urls.py

```diff
@@ -4,28 +4,32 @@
 urlpatterns = [
 
     # GET /login
-    url(r'^login/?$', viewsauth.Login.as_view(), name='login'),
+    url(r'^author/login/?$', viewsauth.Login.as_view(), name='login'),
 
     # POST /logout
-    url(r'^logout/?$', viewsauth.Logout.as_view(), name='logout'),
+    url(r'^author/logout/?$', viewsauth.Logout.as_view(), name='logout'),
 
     # GET POST /author/profile
-    url(r'^profile/?$', viewsauth.AuthorProfile.as_view(), name='profile'),
+    url(r'^author/profile/?$', viewsauth.AuthorProfile.as_view(), name='profile'),
 
     # GET /author/friends/:username
```

Comment:
This is missing /author? Is the new spec correct? If so this comment should be updated.